pages/parent-enrollment.tsx

```diff
@@ -0,0 +1,76 @@
+import Wrapper from "@components/SDCWrapper";
+import { Flex, Button, Center, Text, VStack } from "@chakra-ui/react";
+import React, { useState } from "react";
+import { CloseButton } from "@components/CloseButton";
+
+export default function ParentEnrollClass(): JSX.Element {
+    // Next button is disabled by default, activates when a child is selected
+    const children = ["Christina Ru", "Raewyn Tsai", "Stacy Kwok"];
+    const [selectedChild, setSelectedChild] = useState<string>("");
+
+    return (
+        <Wrapper>
+            <Flex justifyContent="flex-end">
+                {/* navigate to browse programs page instead of going back */}
+                <CloseButton href="/"></CloseButton>
+            </Flex>
+            <Center>
+                <Text align="center" mt="15px" fontWeight="700" fontSize="36px">
+                    Program Registration
+                </Text>
+            </Center>
+            <Center>
+                <Text pb="55px" align="center" mt="30px">
+                    Who would you like to register for{" "}
+                    <b>
+                        Building Bridges with Music - <br />
+                        Singing Monkeys (Ages 9 and under)?{" "}
+                    </b>
+                    (select one)
+                </Text>
+            </Center>
+
+            <Center>
+                <VStack spacing={5}>
+                    {children.map((childName) => {
+                        return (
+                            <Button
+                                backgroundColor="white"
+                                lineHeight="24px"
+                                fontSize="16px"
+                                fontWeight="normal"
+                                textColor="#0C53A0"
+                                borderRadius="6px"
+                                height="50px"
+                                width="340px"
+                                key={childName}
+                                onClick={() => setSelectedChild(childName)}
+                                border={
+                                    selectedChild === childName
+                                        ? "2px solid #0C53A0"
+                                        : "2px solid #E1E1E1"
+                                }
+                            >
```

Comment:
Also, I don't know if we want the box to go grey when you hover and click on it, maybe @christinaruu or @stacykwok can confirm this? I assume not though, in which case you would need to update the `_hover` and `_active` props to maintain the white background.
<img width="387" alt="Screen Shot 2021-08-05 at 10 02 57 PM" src="https://user-images.githubusercontent.com/42329517/128444672-4fd2c5b4-ca32-41c1-8d43-e38b4ad8e056.png">

Reply:
@christinaruu @stacykwok lmk what you think, I thought it was worth trying this out, since it's a pretty easy fix/removal

Reply:
yeah we think it would be better to differentiate the hover and clicked states, thanks for bringing it up! sorry about the changes :( 
here are the final states: https://www.figma.com/file/ad7RNFErTGP7HkL52edTwQ/SDC-External-Application?node-id=1574%3A18497

Reply:
hi @christinaruu , here are the new button states: 
Hover
![image](https://user-images.githubusercontent.com/71860803/128614383-730335c7-09a3-4b7d-962c-ded1738235b6.png)

Clicked/Selected
![image](https://user-images.githubusercontent.com/71860803/128614390-8c077b04-9376-4a43-95ea-f27b56b87529.png)